.github/workflows/update-readme.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
         with:
-          commit-message: Update README
+          commit-message: Update README [skip ci]
```

Comment:
did you test whether this works? We had this message there before but it was not working at that time. We can just try out

Reply:
Hmm, yeah, you're right, this gets stripped on commit. Probably a security measure. Never mind then!